### PR TITLE
Adding description of using postcss-js parser with babel 6 compiler

### PR DESCRIPTION
**Pull request by komlev**
postcss-js parser produce wrapped styles, adding description to fix this problem

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -180,6 +180,11 @@ export default {
 }
 ```
 
+> If you are using babel 6 compiler and up you need to do the following in order for setup to work
```
Comment (michael-ciniawsky) [inline README.md]: - `babel 6 compiler` => `Babel >= v6` 
- in order for setup  => in order for *the* setup to work

File: README.md [hunk 2]
```diff
@@ -180,6 +180,11 @@ export default {
...
+
+> * It is suggested to have only **default** export per style module
```
Comment (michael-ciniawsky) [inline README.md]: only => only *one*  || only => one ?

File: README.md [hunk 3]
```diff
@@ -180,6 +180,11 @@ export default {
...
+> * You need to add [babel-plugin-add-module-exports] to you configuration
```
Comment (michael-ciniawsky) [inline README.md]: Maybe switch order here, but I'm nitpicking 🐤 here 😛 

1. add babel-plugin-add-module-exports
2. only one **default** export per module/file

**Comment by komlev** [comment]
Done

**Pull request merged**